server/api/routes/rooms.js

```diff
@@ -273,15 +273,13 @@ router.post('/join/:roomId', authenticateUser, async (req, res) => {
     - Adds user to room
     - Called by user
 */
-router.post('/add/:roomId', authenticateUser, async (req, res) => {
```

Comment:
we will only support adding a room by joinCode? Does this mean that adding via search results will be grabbing the joinCode rather than roomId